core/controllers/base_test.py

```diff
@@ -888,3 +889,19 @@ def test_no_error_is_raised_on_opening_new_tab_after_signup(self):
         )
 
         self.get_html_response('/about')
+
+
+class GoogleAnalyticsHandlerTests(test_utils.GenericTestBase):
+
+    def test_analytics_handler_returns_required_values(self):
+
+        values_dict = {
```

Comment:
expected_values_dict

Or, you could just inline it, since it's only used once. That would probably be better.

Reply:
Okay

Reply:
Done